src/components/modules/toolbar/toolbox.ts

```diff
@@ -183,12 +183,16 @@ export default class Toolbox extends Module {
     //   return;
     // }
 
-    const userToolboxSettings = this.Editor.Tools.getToolSettings(toolName)[userSettings.TOOLBOX] || {};
+    const userToolboxSettings = this.Editor.Tools.getToolSettings(toolName)[userSettings.TOOLBOX];
+
+    if ((userToolboxSettings ?? toolToolboxSettings) === false) {
```

Comment:
Pls add a jsdoc comment describing this moment

Reply:
Fixed.